composer.json

```diff
@@ -70,6 +70,11 @@
             "docroot/profiles/contrib/{$name}": ["type:drupal-profile"],
             "docroot/themes/contrib/{$name}": ["type:drupal-theme"],
             "drush/contrib/{$name}": ["type:drupal-drush"]
+        },
+        "patches": {
+            "drupal/businessgroup_zymphonies_theme": [
+                "https://www.drupal.org/files/issues/businessgroup_zymphonies_theme-icons_location-2840589-2-8.patch"
```

Comment:
This is not applicable, the patches should be  a key => value pair. The key should be the link to the issue and the value the path to the patch.